_functions/synth_parameters/delay.md

```diff
@@ -2,4 +2,5 @@
 title: delay
 category: synth_parameters
 ---
-a pattern of numbers from 0 to 1. Sets the level of the delay signal.
+a pattern of numbers that set the initial level of the delay signal.  I.e. a value of one means the first echo will be as
```

Comment:
Huh... I didn't realize that!  I thought it still needed to be between 0 and 1.  Hmm... sounds like we need a little set of examples on what to do with strange and interesting values of `delay`.